cmd/do-agent/config.go

```diff
@@ -152,12 +155,19 @@ func initWriter(g gatherer) (metricWriter, throttler) {
 }
 
 func initDecorator() decorate.Chain {
-	return decorate.Chain{
+	chain := decorate.Chain{
 		compat.Names{},
 		compat.Disk{},
 		compat.CPU{},
 		decorate.LowercaseNames{},
 	}
+
+	// If additionalLabels provided convert into decorator
+	if len(config.additionalLabels) != 0 {
```

Comment:
shouldn't this decorator come before the lowercase names one? and possibly  before compat.Names too?

Reply:
I don't think it should matter the order?

Reply:
well if someone adds a label: "SCREAMING_CPU: value",  we won't be normalizing it to lowercase if  they are out of order.

Reply:
we won't be doing that anyways since lowercase only applies to the metric name

Reply:
ah, good point